core/domain/takeout_service.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions to export the data of all user related models from a given
+user_id.
+"""
+from __future__ import absolute_import   # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import re
+
+from core.platform import models
+
+(
+    user_models, collection_models, exploration_models, feedback_models,
```

Comment:
Consider alphabetizing this list, in order to avoid accidental duplicates if it is modified in the future.